styleguide/source/_patterns/01-molecules/08-lists/11-list-events/list-events.twig

```diff
@@ -0,0 +1,46 @@
+<ul class="list-events">
+  <li class="list-events_item">
+    <div class="list-events_item_summary">
+      {% include 'atoms-link-black' with { 'content': 'Lorem Ipsum Dolor Amet Consectetur Elit', 'class' : 'list-events_item_summary_title'} %}
+      {% include 'atoms-paragraph' with { 'content': 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.', 'class' : 'list-events_item_summary_desc'} %}
+    </div>
+    <div class="list-events_item_date">
+       {% include 'atoms-paragraph' with { 'content': 'July 1, 2015', 'class': 'list-events_item_date_day' } %}
```

Comment:
Comp and prod use abbreviated month with period, recommending changing from `July` to `Jul.`